### PR TITLE
Added assignments and evaluation code

### PR DESCRIPTION
**Pull request by preetipie22**

### CODE REVIEW

File: Assignments/HTML_CSS/Assign6/Preeti/PreetiA6_1.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            <input type="radio" id="m" name="male">
+            <label for="m">Male</label>
+            <input type="radio" id="f" name="female">
+            <label for="f">Female</label>
+            <input type="radio" id="o" name="others">
+            <label for="o">Others</label>
+            <p>Age:</p>
+            <input type="radio" id="less" name="age1">
+            <label for="less">less than 18</label>
+            <input type="radio" id="equal" name="age2" checked>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign6/Preeti/PreetiA6_1.html]: check attribute

File: Assignments/HTML_CSS/Assign6/Preeti/PreetiA6_1.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        <div class="end">
+            <input type="submit"  value="Submit">
+        </div>
+
+
+
+    </form>
+</body>
+
+</html>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign6/Preeti/PreetiA6_1.html]: add new line 
remove extra lines

File: Assignments/HTML_CSS/Assign6/Preeti/PreetiA6_1.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    text-align: right;
+    background-color: mediumslateblue;
+    font-style: italic;
+}
+#basicDetails{
+    padding: 20px;
+}
+.end{
+  text-align: center;
+}
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign6/Preeti/PreetiA6_1.css]: add new line

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
indentation needed

File: Assignments/HTML_CSS/Assign6/Preeti/PreetiA6_1.html (hunk 1 above)
Comment (preetipie22) [reply Assignments/HTML_CSS/Assign6/Preeti/PreetiA6_1.html:30]: @Pawan1005 please elaborate which attribute I need to check?

Comment (preetipie22) [reply Assignments/HTML_CSS/Assign6/Preeti/PreetiA6_1.html]: > check attribute

please elaborate which attribute I need to check?

Comment (Pawan1005) [reply Assignments/HTML_CSS/Assign6/Preeti/PreetiA6_1.html]: for checked

**Pull request merged**